android/AndroidManifest.xml

```diff
@@ -7,9 +7,6 @@
   <uses-feature
       android:glEsVersion="0x00020000"
       android:required="true"/>
-  <uses-feature
-      android:name="android.hardware.touchscreen"
-      android:required="true"/>
```

Comment:
Remove only required=true?

This is an important change that may require adding support for non-touchscreen input (remote controls, joysticks, etc.).

Reply:
I can set it to `android:required="false"` because requiring this breaks DeX mode according to the documentation.

I do not see how this forces us to add support for external accessories explicitly as it only allows for non-touchscreen devices. We can officially maintain touch support while deleting this for DeX mode purposes.

Reply:
Now OM claims that the touch input is required to use the app. Removing that claim allows users to install OM on non-touch devices. Where users can not use OM, because there is no touch support.